docs/decisions/0009-moving-catalog-diffing-to-enterprise-catalog.rst

```diff
@@ -0,0 +1,89 @@
+Moving the responsibility of catalog diffing to the enterprise catalog
+----------------------------------------------------------------------
+
+Status
+======
+
+Draft
+
+Context
+=======
+
+Recent, now reverted, attempts to make efficiency improvements to the integrated channels requesting data from the
+enterprise catalogs have highlighted short comings of the current architecture in terms of the capabilities of the
+Integrated Channels to accurately report the last time a customer's full catalog has been successfully updated. Because
+the Integrated Channels are responsible for chunking transmissions (only sending as much as the individual channel's API
+allows at a given time) such that not all content under a customer's catalog is guaranteed to be transmitted in a single
+scheduled run, we cannot accurately determine if all content under a customer's catalog has been updated since the
+catalog has last updated without first using the enterprise catalog's currently available API `get_content_metadata`
+endpoint.
+
+Consequences
+============
+
+**querying all content from the catalog service is expensive**- Asking for all content under the catalog, for every
+customer, for every catalog, on every transmission is not only an undesirably large request to be sending, but also
+consumes much of the enterprise catalog's DB load. We need a smarter solution to how the Integrated Channels requests
+content from the catalog service.
+
+**putting the responsibility of determining differences** between what's already been sent to the customers and what's
+currently in customer's catalogs adds much complexity to the integrated channels which can make it unapproachable from
+an onboarding perspective but also from a debugging point of view. Log noise and number of potential error sources means
+tracking down issues within the Integrated Channels are costly and time consuming.
```

Comment:
Nit: "are costly and time consuming" -> "are time consuming and thus, costly."  (time is money).